src/main/java/peoplesoft/ui/regions/JobCard.java

```diff
@@ -51,10 +51,17 @@
      */
     public JobCard(Job job, int displayedIndex) {
         super(FXML);
+
+        // calculate time
+        int hH = job.getDuration().toHoursPart();
+        int mins = job.getDuration().toMinutesPart();
+        String mM = mins == 0 ? "" : (mins + "m");
+
+        // assign values
         this.job = job;
         idx.setText(displayedIndex + "");
         desc.setText(job.getDesc());
-        duration.setText(job.getDuration().toString());
+        duration.setText(String.format("%dh%s", hH, mM));
```

Comment:
Maybe adding a space or a colon as a visual separator between the hours and minutes would make it more readable?

Reply:
Good suggestion